### ISSUE TITLE
Staging Review finding: Progressive disclosure may be confusing for some users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#138284]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Core526
**Product:** 21-526ez Disability Benefits Application
**Feature:** Improve Evidence Submission 0781 Screens

## Findings details

**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (IA), @allison0034 (Design)

### Description

The use of radio buttons to initiate multi-stage progressive disclosures and/or multiple long forms, may be confusing for some users:
* Users don’t expect radio buttons to trigger large content changes.
* Users cannot see the full scope of the form or what is required
* Multi‑stage reveals break the mental model of “select one option” and instead turn radios into navigation controls.
* Users with cognitive disabilities struggle with hidden, conditional, or cascading content.
* Multi‑stage reveals require remembering previous choices and understanding dependencies.
* Screen reader users don't see the different reveals, so may not understand that radio selection is altering the content of the page

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="551" height="802" alt="radio progressive disclosure" src="https://github.com/user-attachments/assets/4938a472-0385-4c3a-b5ce-cfe82e7e7efd" />

<img width="465" height="541" alt="progressive disclosure" src="https://github.com/user-attachments/assets/706e3a43-2f2e-4f0c-ba8d-a87d9fda7222" />

</details>

## Recommended action

Break apart these interactions into "one thing per page" - https://design.va.gov/patterns/ask-users-for/a-single-response

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 3.2.2 On Input - Level A](https://www.w3.org/TR/WCAG22/#on-input)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
